src/android/Print.java

```diff
@@ -259,6 +259,12 @@ private boolean createReceiptData(Order order) {
 			if (order.customer_address != null && !order.customer_address.isEmpty())
 				builder.append("Address: " + order.customer_address + "\n");
 
+			if (order.handoff_provider != null && !order.handoff_provider.isEmpty())
```

Comment:
Is this a display formatted value, like DoorDash, or the system value, like doordash?
We should make sure it's formatted nicely